frontend/coprs_frontend/run/check_for_anitya_version_updates.py

```diff
@@ -141,33 +122,63 @@ def package_from_source(backend, source_json):
         raise Exception('Unsupported backend {0} passed as command-line argument'.format(args.backend))
 
 
+def is_prerelease(version: str) -> bool:
+    """
+    Detect a pre-release version string.
+    """
+    known_suffixes = [
+        ".dev",
+        ".beta",
+        ".rc",
+        ".alpha",
```

Comment:
@befeleme, do you think this is enough for now?

Reply:
In the build data I see often "a" too (eg. `openapi-core 0.17.0a1`). The remaining look fine.